log/gps/gps_destinator_trk.ksy

```diff
@@ -0,0 +1,82 @@
+meta:
+  id: gps_destinator_trk
+  title: Destinator track
+  application: Destinator
+  file-extension:
+    - dat
+  license: GPL-2.0-or-later
+  encoding: utf-16le
+  endian: le
+
+doc: |
+  Destinator formats
```

Comment:
Like above, this should have at least a short explanation of the software and format.